settings/provisioner/topology/ceph.yml

```diff
@@ -7,11 +7,13 @@ os:
     variant: "rhel7"
 disks:
     disk1: &disk1
+        import_url: null
```

Comment:
No. We don't need to define. I've added that just to indicate that import_url can be defined.